### PR TITLE
Remove system-probe bundled files from git

### PR DESCRIPTION
**Pull request by brycekahle**
### What does this PR do?

Remove system-probe bundled files from git. This leaves `bindata.go`, which provides the Asset API to prevent problems building/running go tools.

### Motivation

These files are re-generated on build. They currently cause extra work to deal with them whenever parallel branches modify any ebpf code.

### CODE REVIEW

**Review by leeavital — APPROVED** [review]

**Review by mx-psi — APPROVED** [review]

**Review by hmahmood — APPROVED** [review]

**Comment by hmahmood** [comment]
Related to: https://github.com/DataDog/datadog-agent/pull/7177

File: .gitignore [hunk 1]
```diff
@@ -121,6 +121,8 @@ tools/windows/install-help/cal/packages/
 
 # ebpf object files
 pkg/ebpf/bytecode/build/
+pkg/ebpf/bytecode/bindata/
+!pkg/ebpf/bytecode/bindata/bindata.go
```
Comment (hmahmood) [inline .gitignore:125]: Should we remove this file also since it is generated?

Comment (brycekahle) [reply .gitignore:125]: We would have to completely remove bundling then.  Without that file, even simple commands like `inv -e deps` fail.

Comment (brycekahle) [reply .gitignore:125]: Nevermind. I'm not sure what the problem was.

**Comment by brycekahle** [comment]
Closing in favor of #7177

**Pull request closed without merging**